### ISSUE TITLE
Gown GraphQL queries on gown landing page (/gowns)

### ISSUE THREAD

**Issue by jserrao** [jserrao/modern-trousseau-gatsby#151]
Each collection of gowns should show top 3 gowns.

**Comment by jserrao** [comment]
@mdrisco4 - did you get the queries into this page? Do you need help?

**Comment by jserrao** [comment]
I think you're also flipping the mobile/desktop carousel use cases on this one, correct @mdrisco4 ?

**Comment by mdrisco4** [comment]
@jserrao Yes.  I think I was a bit overzealous on getting a PR ready for this by tonight with job materials on my plate.  This is my first order of business.  I will get as far on this as I can tonight and aim for a PR for the desktop/mobile flipping sometime tomorrow.  After that I can get back to GraphQL queries.

**Comment by mdrisco4** [comment]
@jserrao I worked out the logic today with regards to making the desktop version a carousel and just need to finish flipping all the styling, functionality and media queries so that there are carousels for individual collections stacked on top of each other that are on the desktop version (4 collections on the page) and the mobile version is effectively a page encompassing carousel that works with arrows at the top of the page that cycle through collections with 8 images of gowns from each collection per the look from the Figma mockup.  When I am at the point of populating the mapped data onto the pages I am just wondering about details with the dynamics.  On the desktop version I can just have it set up so that the carousels cycle through all gowns for a given collection...however, with the 8 image mobile version I am wondering how I should approach that build (i.e.  should I just fetch the first 8 gowns in a given collection?).  Please advise.  I am hoping I will have the 'Gowns Page' ready for a PR by weeks end, at which point we can discuss the issue with the 'Collections Page' that you had mentioned . "--@mdrisco4 - we should probably talk this over before you dive in. We need to dynamically generate collections as there will be 1,2,3.....n of them."  Sorry this is so wordy...just trying to get as many details as I can before moving forward so I don't have to blow your shit up every time I have an inquiry.  Thx
